charts/fluentd/templates/_pod.tpl

```diff
@@ -14,8 +15,15 @@ containers:
   - name: {{ .Chart.Name }}
     securityContext:
       {{- toYaml .Values.securityContext | nindent 6 }}
+    {{- if eq .Values.output.type "custom" }}
+    image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default $defaultTagCustom }}"
+    {{- else }}
     image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default $defaultTag }}"
+    {{- end }}
     imagePullPolicy: {{ .Values.image.pullPolicy }}
+  {{- if and (.Values.output.plugins.enabled) (gt (len .Values.output.plugins.pluginsList) 0) }}
+    command: ["/bin/sh", "-c", "/fluentd/etc/config.d/install-plugins.sh"]
```

Comment:
```suggestion
    command:
    - "/bin/sh"
    - "-c"
    - |
       #!/bin/sh
     {{- range $plugin := .Values.output.plugins.pluginsList }}
       fluent-gem install {{ $plugin }}
     {{- end }}
      exec 
```

Maybe we can add this here (indentation might be off) and get ride of the configmap?